src/corehost/common/pal.h

```diff
@@ -123,6 +125,8 @@ namespace pal
     bool utf8_palstring(const std::string& str, pal::string_t* out);
     bool pal_clrstring(const pal::string_t& str, std::vector<char>* out);
     bool clr_palstring(const char* cstr, pal::string_t* out);
+
+    inline bool can_allow_signed_dotnet_old_hostfxr() { return false; }
```

Comment:
Hopefully, this difference between Win/Non-Win is temporary and we will remove this function altogether, right?